foundations/javascript_basics/variables_and_operators.md

```diff
@@ -1,86 +1,128 @@
 ### Introduction
 
-In the previous sections you learnt how to structure webpages with HTML and style them with CSS. The next step is to make the webpage *interactive*, which is exactly what JavaScript is for. In this section, we will focus on the fundamentals of JavaScript and how you can use it to manipulate all the various interactions between the web page and user.
+In the previous sections you learned how to structure webpages with HTML and style them with CSS. The next step is to make the webpage *interactive*, which is exactly what JavaScript is for.
+
+In this section, we will focus on the fundamentals of JavaScript and how you can use it to manipulate all the various interactions between the web page and user.
 
 ### Lesson overview
 
 This section contains a general overview of topics that you will learn in this lesson.
 
-- How do you declare a variable?
-- What are three different ways to declare a variable?
-- Which one should you use when?
-- What are the rules for naming variables?
-- What are operators, operands, and operations?
-- What is concatenation and what happens when you add numbers and strings together?
-- What are the different types of operators in JavaScript?
-- What is the difference between == and ===?
-- What are operator precedence values?
-- What are the increment/decrement operators?
-- What is the difference between prefixing and postfixing them?
-- What are assignment operators?
-- What is the Unary Plus Operator?
+- Discuss the different ways you can declare variables.
+- Learn how to write mathematical expressions and the basic arithmetic operators.
+- What is operator precedence and some more advanced operators.
```

Comment:
I'm not sure these three bullet points sufficiently encapsulate what this lesson is about. I do not like the original either though. Can you try this again but be more comprehensive? Also, let's avoid starting these bullets with the word "what" - it just doesn't fit.

Reply:
I changed the wording let me know what you think of it now!